common/landed_titles/01_landed_titles.txt

```diff
@@ -25467,6 +25492,16 @@ e_lordaeron = {
 			}
 			c_undernelly = {
 				color = { 112 62 121 }
+				
+				cultural_names = {
+					name_list_amani = cn_shadeglen
+					name_list_blood_troll = cn_shadeglen
+					name_list_dark_troll = cn_shadeglen
+					name_list_drakkari = cn_shadeglen
+					name_list_farraki = cn_shadeglen
+					name_list_gurubashi = cn_shadeglen
+					name_list_zandalari = cn_shadeglen
+				}
```

Comment:
Why would both c_ferry and c_undernelly use the same cultural name?

Reply:
but they don't? Ferry is Zeb'rokh. Undernelly is Shadeglen.

Reply:
oh sorry I meant c_dawning_isles

Reply:
Dawning Isles don't have a cn. Duchy of Lordamere is Shadeglen (d). County of Ferry is Zeb'rokh. County of Undernelly is Shadeglen (c). County of Valganfield is Shad'watha.